index.html

```diff
@@ -97,6 +97,23 @@
       .usa-stripe:nth-child(odd){
         background-color: #FFFFFF;
       }
+      #japan{
+        display: grid;
+        background-color: #FFFFFF;
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-template-rows: 1fr 1fr 1fr;
```

Comment:
just curious, does it look the same if you take these two lines out?  I have a feeling it would -- let me know!